spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/core/ClientManager.java

```diff
@@ -68,6 +69,13 @@ public interface ClientManager<T, C> extends SmartLifecycle, MqttComponent<C> {
 	 */
 	boolean removeCallback(ConnectCallback connectCallback);
 
+	/**
+	 * Return the managed clients isConnected.
+	 * @return the managed clients isConnected.
+	 * @since 6.4
+	 */
+	boolean isConnection();
```

Comment:
I think the name of the method is slightly wrong.
I believe it has to be `isConnected()`.